### ISSUE TITLE
[Bug]: PWA - Fullscreen button not visible in portrait on mobile

### ISSUE THREAD

**Issue by Nonononoki** [MarmadileManteater/FreeTubeCordova#298]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

The full screen button in the video player is not visible on narrow screen, such a phones in portrait mode.

### Expected Behavior

Full screen button should always be visible on any device.

### Issue Labels

accessibility issue

### FreeTube Version

v0.19.1.109 Beta

### Operating System Version

Android 13

### Installation Method

PWA (Progressive Web App)

### Primary API used

Local API

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by MarmadileManteater** [comment]
I'm not going to get to this immediately, and the fix is, probably, going upstream since its a responsiveness issue.

The fix I'm thinking of would be to just move / remove some of the buttons in the controls. Maybe, the video progress timestamp doesn't need to be shown under a certain width. Maybe, the P-in-P button shouldn't be in the bar, etc.

The only reason this isn't a problem in the app is because it doesn't support P-in-P. When it does, this will also be an issue with the app.

**Comment by MarmadileManteater** [comment]
This was fixed last release.
